s3contents/genericmanager.py

```diff
@@ -292,6 +303,44 @@ def is_hidden(self, path):
         self.log.debug("S3contents.GenericManager.is_hidden '%s'", path)
         return False
 
+    post_save_hook = Any(
```

Comment:
Could you move this to the start of the class definition?

Reply:
Will do